WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -2080,6 +2095,12 @@ - (UIPresentationController *)presentationControllerForPresentedViewController:(
     return nil;
 }
 
+#pragma mark - UIAdaptivePresentationControllerDelegate
+
+- (void)presentationControllerWillDismiss:(UIPresentationController *)presentationController {
+    [self.tableView deselectSelectedRowWithAnimation:YES];
```

Comment:
What do you think adding a check to make this code only runs on the "Site Settings" modal, something like ` if (presentationController.presentedViewController.viewControllers.first is SiteSettingsViewController) { ... }`?

Reply:
Updated. I agree – better be precise.